node/src/components/linear_chain_fast_sync.rs

```diff
@@ -1,617 +0,0 @@
-//! Fast linear chain synchronizer.
```

Comment:
What happened here? I see `linear_chain_fast_sync.rs` as both removed and added... and added with just one line (to which I can't even attach a comment)? Normally removal + addition = modification, but not here for some reason, and there's strange content - is that a mistake?

Reply:
For now, maintaining `linear_chain_fast_sync.rs` has been too great a burden.

`linear_chain_sync.rs` isn't compatible with changes to the wire format (such as in this PR).  We're going to just replace `linear_chain_sync.rs` with fast sync.

Reply:
> linear_chain_sync.rs isn't compatible with changes to the wire format (such as in this PR). We're going to just replace linear_chain_sync.rs with fast sync.

It can't _replace_ it b/c we want to have both abilities – to sync the whole chain for historical/archival purposes and a fast-sync version.